snmp.yml

```diff
@@ -1565,3 +1565,946 @@ fortinet_fortigate:
+          oid: 1.3.6.1.4.1.14988.1.1.7.4
+    - name: mtxrHlCpuTemperature
+      oid: 1.3.6.1.4.1.14988.1.1.3.6
+      indexes:
+        - labelname: mtxrSerialNumber
+          type: OctetString
+      lookups:
+        - labels: [mtxrSerialNumber]
+          labelname: mtxrSerialNumber
+          oid: 1.3.6.1.4.1.14988.1.1.7.3
+        - labels: [mtxrSerialNumber]
+          labelname: mtxrFirmwareVersion
+          oid: 1.3.6.1.4.1.14988.1.1.7.4
+    - name: mtxrHlBoardTemperature
+      oid: 1.3.6.1.4.1.14988.1.1.3.7
+      indexes:
+        - labelname: mtxrSerialNumber
+          type: OctetString
+      lookups:
+        - labels: [mtxrSerialNumber]
+          labelname: mtxrSerialNumber
+          oid: 1.3.6.1.4.1.14988.1.1.7.3
+        - labels: [mtxrSerialNumber]
+          labelname: mtxrFirmwareVersion
+          oid: 1.3.6.1.4.1.14988.1.1.7.4
+    - name: mtxrHlVoltage
+      oid: 1.3.6.1.4.1.14988.1.1.3.8
+      indexes:
+        - labelname: mtxrSerialNumber
+          type: OctetString
+      lookups:
+        - labels: [mtxrSerialNumber]
+          labelname: mtxrSerialNumber
+          oid: 1.3.6.1.4.1.14988.1.1.7.3
+        - labels: [mtxrSerialNumber]
+          labelname: mtxrFirmwareVersion
+          oid: 1.3.6.1.4.1.14988.1.1.7.4
+    - name: mtxrHlActiveFan
+      oid: 1.3.6.1.4.1.14988.1.1.3.9
+      indexes:
+        - labelname: mtxrSerialNumber
+          type: OctetString
+      lookups:
+        - labels: [mtxrSerialNumber]
+          labelname: mtxrSerialNumber
+          oid: 1.3.6.1.4.1.14988.1.1.7.3
+        - labels: [mtxrSerialNumber]
+          labelname: mtxrFirmwareVersion
+          oid: 1.3.6.1.4.1.14988.1.1.7.4
+    - name: mtxrLicUpgrUntil
+      oid: 1.3.6.1.4.1.14988.1.1.4.2
+      indexes:
+        - labelname: mtxrLicSoftwareId
+          type: OctetString
+      lookups:
+        - labels: [mtxrLicSoftwareId]
+          labelname: mtxrLicVersion
+          oid: 1.3.6.1.4.1.14988.1.1.4.4
+        - labels: [mtxrLicSoftwareId]
+          labelname: mtxrLicSoftwareId
+          oid: 1.3.6.1.4.1.14988.1.1.4.1
+    - name: mtxrLicLevel
+      oid: 1.3.6.1.4.1.14988.1.1.4.3
+      indexes:
+        - labelname: mtxrLicSoftwareId
+          type: OctetString
+      lookups:
+        - labels: [mtxrLicSoftwareId]
+          labelname: mtxrLicVersion
+          oid: 1.3.6.1.4.1.14988.1.1.4.4
+        - labels: [mtxrLicSoftwareId]
+          labelname: mtxrLicSoftwareId
+          oid: 1.3.6.1.4.1.14988.1.1.4.1
+    - name: mtxrLicUpgradableTo
+      oid: 1.3.6.1.4.1.14988.1.1.4.5
+      indexes:
+        - labelname: mtxrLicSoftwareId
+          type: OctetString
+      lookups:
+        - labels: [mtxrLicSoftwareId]
+          labelname: mtxrLicVersion
+          oid: 1.3.6.1.4.1.14988.1.1.4.4
+        - labels: [mtxrLicSoftwareId]
+          labelname: mtxrLicSoftwareId
+          oid: 1.3.6.1.4.1.14988.1.1.4.1
+
+    - name: mtxrNeighborIfIndex
+      oid: 1.3.6.1.4.1.14988.1.1.11.1.1.8
```

Comment:
I can't find this particular entry in the MIB, however the index is mtxrNeighborTable and there should be no lookups.